calculator-server/controllers/calculator-stateless.js

```diff
@@ -0,0 +1,67 @@
+const Calculator = require("../../core/calculator");
+
+/*
+{
+    “operand1”: 1;
+    “operand2”: 24;
+    “operation”: “add”;
+}
+*/
+const answer = (req, res) => {
+  console.log(req.body);
+  const { operand1, operand2, operation } = req.body;
+
+  const calculator = new Calculator();
+  calculator.add(operand1);
+
+if (operation === undefined){
+  res.status(400).json({ error: "An operation is needed" });
```

Comment:
you should have a return statement after this line to exit the function and not continue trying to process the request